CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.3
+
+* Fix: explicitly specify `'localhost'` for round trip tests
```

Comment:
Reading this again, this is just a test.
I think we can skip publishing this in a version for itself.

Reply:
Sure, reverted.